include/swift/Sema/CSBindings.h

```diff
@@ -541,6 +544,14 @@ class BindingSet {
     return x.isPotentiallyIncomplete() < y.isPotentiallyIncomplete();
   }
 
+  /// Return string for atomic literal kinds (integer, string, & boolean) for
+  /// printing in debug output.
+  StringRef getAtomLiteral(ExprKind EK) const;
```

Comment:
Both this and `getCollectionLiteral` could use `AsString` in the name and return `std::string`. Don't forget that `StringRef` doesn't actually manage the underlying storage.

Reply:
Also, should this be static? I remember we discussing that :)

Reply:
Yeah